lib/core/src/lib/pagination/pagination.component.html

```diff
@@ -17,6 +17,7 @@
         <button
             mat-icon-button
             [attr.aria-label]="'CORE.PAGINATION.ARIA.ITEMS_PER_PAGE' | translate"
+            [attr.title]="'CORE.PAGINATION.ARIA.ITEMS_PER_PAGE' | translate"
```

Comment:
I don't think that `title` provides the visible label text especially when `label` is already there.. Please check out https://www.w3.org/WAI/WCAG21/Techniques/failures/F96 there might be other issue that are causing this error